Food.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang = "en">
+<head>
+    <meta charset = "UTF-8">
+    <meta http-equiv = "X-UA-Compatible" content = "IE=edge">
+    <meta name = "viewport" content = "width=device-width, initial-scale=1.0">
+    <link rel = "stylesheet" href = "Food.css">
+    <title>Catering Website</title>
+</head>
+<body>
+    <header>
+        <div class = "logo">
+            <p>Gourmet au Catering</p>
+        </div>
+        <nav>
+            <ul>
+                <li>About</li>
+                <li>Menu</li>
+                <li>Contact</li>
+            </ul>
+        </nav>
+    </header>
+    <section class = "banner">
+        <img src = "/Users/lakshyagoel/Downloads/GitHub/Gourmae Website/hamburger.jpg" alt = "Hamburger">
+        <p class = "btitle">Le Catering</p>
+    </section>
+    <section class = "about">
+        <div class = "left">
+            <img src = "/Users/lakshyagoel/Downloads/GitHub/Gourmae Website/tablesetting2.jpg" alt = "Table">
+        </div>
+        <div class = "right">
+            <p class = "title">About Catering</p>
+            <p class = "subtitle">Tradition Since 1889</p>
+            <p class = "para1">The Catering was founded in blabla by Mr. Smith in Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Guis aute iruredolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. We only use <span style = "background-color:rgb(226,226,226)">seasonal</span> ingrediants.</p><br>
+            <p class = "para2">Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia doserunt mollit anim id est laborum consectetur adipiscing elit, sed do eiusmod temporincididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+        </div>
+    </section>
```

Comment:
Menu section is missing.

Reply:
Menu was not given in the instructions section. So I didn't included that.